tutorials/7-cosmjs/3-multi-msg.md

```diff
@@ -158,7 +158,7 @@ Confirm this by making the change in your `experiment.ts` from the previous sect
 
 <HighlightBox type="best-practice">
 
-Building a transaction in this way is recommended. `SigningStargateClient` offers you convenience methods such as `sendTokens` for simple use cases, and to demonstrate how to build messages.
+Building a transaction in this way is recommended. `SigningStargateClient` offers you convenient methods such as `sendTokens` for simple use cases, and to demonstrate how to build messages.
```

Comment:
A convenience function / method [is a thing ;)](https://softwareengineering.stackexchange.com/questions/272810/what-is-the-difference-between-a-helper-and-a-convenience-function).

```suggestion
Building a transaction in this way is recommended. `SigningStargateClient` offers you convenience methods such as `sendTokens` for simple use cases, and to demonstrate how to build messages.
```

Reply:
I apologise! XD